src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -107,8 +108,20 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $container = new ContainerBuilder();
+        $willBeAvailable = static function (string $package, string $class, string $parentPackage = null) use ($container) {
+            $parentPackages = (array) $parentPackage;
+            $parentPackages[] = 'symfony/framework-bundle';
+
+            return $container->willBeAvailable($package, $class, $parentPackages);
```

Comment:
should the `willBeAvailable` method really be defined in `ContainerBuilder` as an instance method ? This leads to such weird cases where we have to create a ContainerBundle just to use it. And AFAICT, it does not really depend on other parts of the ContainerBuilder.

Reply:
I thought the same, and I think it's important to discourage using the method outside of any compilation steps. By making it a non-static method on ContainerBuilder, we kinda achieve this IMHI.

Reply:
`Compiler::isAvailable` could work

Reply:
I made the method static instead.